src/main/scala/io/iohk/ethereum/db/dataSource/RocksDbDataSource.scala

```diff
@@ -160,10 +158,10 @@ class RocksDbDataSource(
     * This function closes the DataSource, without deleting the files used by it.
     */
   override def close(): Unit = {
-    logger.debug(s"About to close DataSource in path: ${rocksDbConfig.path}")
+    log.info(s"About to close DataSource in path: ${rocksDbConfig.path}")
     assureNotClosed()
```

Comment:
I wonder if we should not check it after locking ?

Reply:
For this case, it is the same. I changed it anyway, and put the same behavior that the other functions.